dll/win32/shell32/CDefView.cpp

```diff
@@ -539,11 +544,12 @@ void CDefView::CheckToolbar()
     }
 }
 
-void CDefView::UpdateStatusbar()
+void CDefView::UpdateStatusbarInner()
```

Comment:
Using standard naming would be:
```suggestion
void CDefView::UpdateStatusbarWorker()
```
that would be called from the thread function.

Reply:
Renamed. Added parameter `hThread`.